src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
+import java.util.Random;
 
 public class Application {
     public static void main(String[] args) {
+
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        System.out.println(ball1.toString());
+        System.out.println(ball2.toString());
+        System.out.println(ball3.toString());
```

Comment:
Instead of calling `toString()` explicitly, you can just pass the `ball` object to the `System.out.println` method, and `toString()` will be called implicitly. This makes the code cleaner.